### ISSUE TITLE
Uncaught TypeError: Cannot read property 'childNodes' of undefined

### ISSUE THREAD

**Issue by mreif** [nicolas-mosch/java-bytecode-disassembler#6]
[Enter steps to reproduce:]

1. select a Java 10 class file  -> show 3/addres code

**Atom**: 1.28.2 x64
**Electron**: 2.0.5
**OS**: Mac OS X 10.13.6
**Thrown From**: [Java-Bytecode-Disassembler](https://github.com/nicolas-mosch/java-bytecode-disassembler) package 0.12.4

### Stack Trace

Uncaught TypeError: Cannot read property 'childNodes' of undefined

```
At /Users/mreif/.atom/packages/Java-Bytecode-Disassembler/lib/java-bytecode-disassembler.js:115

TypeError: Cannot read property 'childNodes' of undefined
    at HTMLElement.javaTacProcessMethod (/packages/Java-Bytecode-Disassembler/lib/java-bytecode-disassembler.js:115:84)
    at CommandRegistry.handleCommandEvent (/private/var/folders/gr/yxt2jmkx6nj3p92twmnc3f040000gr/T/AppTranslocation/EBB0A6BC-21E4-4D1D-8427-E84714464B8F/d/Atom.app/Contents/Resources/app/src/command-registry.js:384:49)
    at CommandRegistry.dispatch (/private/var/folders/gr/yxt2jmkx6nj3p92twmnc3f040000gr/T/AppTranslocation/EBB0A6BC-21E4-4D1D-8427-E84714464B8F/d/Atom.app/Contents/Resources/app/src/command-registry.js:273:23)
    at AtomEnvironment.dispatchContextMenuCommand (/private/var/folders/gr/yxt2jmkx6nj3p92twmnc3f040000gr/T/AppTranslocation/EBB0A6BC-21E4-4D1D-8427-E84714464B8F/d/Atom.app/Contents/Resources/app/src/atom-environment.js:1351:25)
    at EventEmitter.outerCallback (/private/var/folders/gr/yxt2jmkx6nj3p92twmnc3f040000gr/T/AppTranslocation/EBB0A6BC-21E4-4D1D-8427-E84714464B8F/d/Atom.app/Contents/Resources/app/src/application-delegate.js:346:53)
    at emitThree (events.js:136:13)
    at EventEmitter.emit (events.js:217:7)
```

### Commands

```
  3x -4:37.9.0 core:move-down (div.tool-panel.tree-view)
  3x -4:37.3.0 core:move-up (div.tool-panel.tree-view)
     -4:36.6.0 tree-view:collapse-directory (div.tool-panel.tree-view)
     -3:36.6.0 java-tac:process-method (span.file.icon.icon-file-text)
     -3:31.2.0 core:cancel (li.list-item.entry)
  4x -3:21.3.0 core:move-down (div.tool-panel.tree-view)
     -3:20.7.0 tree-view:expand-item (div.tool-panel.tree-view)
  2x -3:20.4.0 core:move-down (div.tool-panel.tree-view)
     -3:19.6.0 tree-view:expand-item (div.tool-panel.tree-view)
     -3:19.2.0 core:move-down (div.tool-panel.tree-view)
     -3:19.1.0 tree-view:expand-item (div.tool-panel.tree-view)
     -3:18.8.0 core:move-down (div.tool-panel.tree-view)
     -3:18.6.0 tree-view:expand-item (div.tool-panel.tree-view)
     -3:18.3.0 core:move-down (div.tool-panel.tree-view)
     -3:17.8.0 tree-view:expand-item (div.tool-panel.tree-view)
     -3:13.2.0 java-tac:process-method (li.file.entry.list-item.status-ignored.selected)
```

### Non-Core Packages

```
Java-Bytecode-Disassembler 0.12.4 
```

**Comment by Rusi1R** [comment]
This happens if you click on "Show 3-Address-Code (Method)" (which is for methods only) when clicking on a .class file.

Did you try using the "Show 3-Address-Code" option in the context menu right below "Opal-Disassemble"?

**Comment by Rusi1R** [comment]
Should be fixed in v0.12.5

The "3-address-code for methods" option is only shown in .class files now.

If this problem still occurs, please open a new issue.
